wp-content/themes/eighshift-task/src/blocks/custom/image-text/image-text.js

```diff
@@ -0,0 +1,109 @@
+import { Fragment } from '@wordpress/element';
+import { InspectorControls, BlockControls } from '@wordpress/editor';
+
+import { getActions } from 'EighshiftBlocksGetActions';
+import manifest from './manifest.json';
+
+import { ImageTextEditor } from './components/image-text-editor';
+import { ImageTextOptions } from './components/image-text-options';
+
+import { ImageOptions } from '../../components/image/components/image-options';
+import { HeadingOptions } from '../../components/heading/components/heading-options';
+import { ParagraphOptions } from '../../components/paragraph/components/paragraph-options';
+import { ButtonOptions } from '../../components/button/components/button-options';
+
+import { HeadingToolbar } from '../../components/heading/components/heading-toolbar';
+import { ParagraphToolbar } from '../../components/paragraph/components/paragraph-toolbar';
+
+export const ImageText = (props) => {
+  const {
+    setAttributes,
+    attributes: {
+      mediaPosition,
+      includeBtn,
+      headingStyleColor,
+      headingStyleSize,
+      paragraphStyleColor,
+      paragraphStyleSize,
+      btnUrl,
+      btnStyleSize,
+      btnStyleColor,
+      btnStyleSizeWidth,
+      btnId,
+
+      headingStyleAlign,
+      headingLevel,
+      paragraphStyleAlign,
+    },
+  } = props;
+
+  // Get all auto generated attribute change actions, spread them and
+  // add manually defined attribute change actions/methods
+  const actions = {
+    ...getActions(props, manifest),
+    onChangeMedia: (value) => {
+      setAttributes({
+        mediaId: value.id,
+        mediaUrl: value.url,
+      });
+    },
+  };
+
+  return (
+    <Fragment>
+      <InspectorControls>
+        <ImageTextOptions
```

Comment:
this is a block component just pass all the attributes and actions.

Reply:
comment before in attributes pass only attributes like this.

attributes={attributes}